docs/understanding-tracking-design/create-tracking-plan-with-tracking-scenarios/managing-tracking-scenarios-console/index.md

```diff
@@ -0,0 +1,168 @@
+---
+title: "Managing tracking scenarios in the console"
+sidebar_label: "🆕 Managing tracking scenarios in the console"
```

Comment:
Technically I guess having it in the parent page is sufficient, but feel free to keep here as well. (Do use #509 when it’s ready.)

Reply:
Also, this label is very long. I am trying to keep the sidebar less bulky. Maybe something like “Tracking scenarios in the UI”?

Reply:
Done